pkg/3scale/amp/operator/backend_reconciler.go

```diff
@@ -216,13 +220,16 @@ func Backend(apimanager *appsv1alpha1.APIManager, client client.Client) (*compon
 	return component.NewBackend(opts), nil
 }
 
-func GetBackendRedisSecret(apimanagerNs string, ctx context.Context, client client.Client) (string, string) {
+func GetBackendRedisSecret(apimanagerNs string, ctx context.Context, client client.Client) (string, string, bool, bool) {
 	backendRedisSecret := &v1.Secret{}
 	client.Get(ctx, types.NamespacedName{
```

Comment:
```suggestion
	err := client.Get(ctx, types.NamespacedName{
```
We should add error handling here to be safe.